via/templates/pdf_viewer.html.jinja2

```diff
@@ -10,12 +10,22 @@
       the rest of via, however, we serve it with the PDF.js viewer application.
     -#}
     <link rel="canonical" href="{{ url }}"/>
+
+    <link rel="preload" as="fetch" href="{{ pdf_url }}" crossorigin>
+    <link rel="preload" as="worker" href="../build/pdf.worker.js">
+    <link rel="preload" as="script" href="{{ client_embed_url }}">
```

Comment:
So we can start loading the PDF itself, PDF.js's `pdf.worker.js`, and the Hypothesis client, ASAP rather than waiting until the page needs them :+1: